packages/images-plugin/lib/finder.js

```diff
@@ -287,10 +287,10 @@ export default class ImageElementFinder {
    * @returns {Promise<Screenshot & {scale?: ScreenshotScale}>} base64-encoded screenshot and ScreenshotScale
    */
   async getScreenshotForImageFind(screenWidth, screenHeight) {
-    if (!this.driver.getScreenshot) {
+    if (!this.driver?.getScreenshot) {
```

Comment:
if driver is null then the error below won't make much sence

Reply:
true. I'll change this